File: src/c/rm.c
#include <stdio.h>

int main(int argc, char** argv)
{
    if (argc<2)
    {
        printf("Error: need more arguments!\nUsage: rm \"File Name\"");
    }else{
        for (int i = 0; i < argc; ++i)
        {
            if (i>0)
            {
                remove(argv[i]);
            }
        }
    }
    return 0;
}

File: src/cs/rm.cs
using System;
using System.IO;

class rm
{
    static void Main(string[] args)
    {
        if (args.Length < 1)
        {
            Console.WriteLine("Error: need more arguments!\nUsage: rm \"File Name\"");
        }
        else
        {
            for (int i = 0; i<args.Length; i++)
            {
                if (File.Exists(args[i]) == false)
                {
                    Console.WriteLine("Error: file {0} not found!", args[i]);
                }
                else
                {
                    File.Delete(args[i]);
                }
                
            }

        }
    }
}